src/antd-theme/my-antd-theme.less

```diff
@@ -86,3 +86,7 @@
 @label-required-color: #f43f5e;
 // steps
 @steps-background: transparent;
+// switch
+@disabled-color: #475569;
```

Comment:
Have you checked if that's working OK with all the UI?

Reply:
I agree this is a very generic variable. Checked code and also App. We already overwrote the once we where interested in (e.g. `@btn-disable-color`, `@input-disabled-color` etc). moves to the top of the file for better visibility